packages/analytics-types/src/element-interactions.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from './logger';
+
+export type ActionType = 'click' | 'change';
+
+/**
+ * Default CSS selectors to define which elements on the page to track.
+ * Extend this list to include additional elements to track. For example:
+ * ```
+ * autocapturePlugin({
+ *    cssSelectorAllowlist: [...DEFAULT_CSS_SELECTOR_ALLOWLIST, ".my-class"],
+ * })
+ * ```
+ */
+export const DEFAULT_CSS_SELECTOR_ALLOWLIST = [
```

Comment:
Can this be exported from the `plugin-autocapture-browser` instead? That way we don't have to update it in two places.

Reply:
`analytics-types` is a dependency of `plugin-autocapture-browser`. It's a circular dependency if we import these from `plugin-autocapture-browser` in `analytics-types`.

Reply:
Sounds good, we can think of some way to resolve this is the next version

Reply:
If we keep those in analytics-types, we don’t have circular dependency and this is the pattern other packages, for example page view tracking plugin.